lib/CL/devices/vulkan/pocl-vulkan.c

```diff
@@ -2085,6 +2073,23 @@ pocl_vulkan_read_rect (void *data,
                       size_t const host_row_pitch,
                       size_t const host_slice_pitch)
 {
+  POCL_ABORT_UNIMPLEMENTED ("pocl_vulkan_read_rect is not implemented\n");
+}
+
+void
+pocl_vulkan_write_rect (void *data,
+                      const void *__restrict__ host_ptr,
+                      pocl_mem_identifier * dst_mem_id,
+                      cl_mem dst_buf,
+                      const size_t *__restrict__ const buffer_origin,
+                      const size_t *__restrict__ const host_origin,
+                      const size_t *__restrict__ const region,
+                      size_t const buffer_row_pitch,
+                      size_t const buffer_slice_pitch,
+                      size_t const host_row_pitch,
+                      size_t const host_slice_pitch)
+{
+  POCL_ABORT_UNIMPLEMENTED ("pocl_vulkan_write_rect is not implemented\n");
```

Comment:
...for the vulkan driver. (same for others?)

Reply:
I mean, vulkan is in the function name. Maybe it should be something like "clEnqueueWriteRect is not implemented for the Vulkan driver” or something like that?